### PR TITLE
Added support for parsing new dashboard.

### PR DESCRIPTION
**Pull request by baltimoron**
Parser for the new dashboard and a few minor fixes for related issues.

### CODE REVIEW

File: pkg/bingDashboardParser.py [hunk 1]
```diff
@@ -51,7 +54,8 @@ class Col:
         RE_EARN_CREDITS_PASS = (1,    EARN_CREDITS_RE,
                             "Get the best of Bing by signing in with Facebook.", True,  Action.PASS)
         RE_EARN_CREDITS      = (2,    EARN_CREDITS_RE,                     None, True,  Action.HIT)
-        SEARCH_MOBILE        = (3,    "Mobile search",                     None, False, Action.SEARCH)
+        #SEARCH_MOBILE        = (3,    "Mobile search",                     None, False, Action.SEARCH)
```
Comment (amayer5125) [inline pkg/bingDashboardParser.py]: If we are going to comment this out we should just remove it.

File: pkg/bingDashboardParser.py [hunk 2]
```diff
@@ -101,124 +105,148 @@ def parseDashboardPage(page, bing_url):
...
+                if currentTitle in allTitles:
+                    #already have this reward, skip it
+                    continue
+                else:
+                    newRwd = Reward()
+                    allTitles.add(currentTitle)
+                    validRwd = createRewardNewFormat(page, currentTitle, newRwd)
+                    if validRwd:
+                       allRewards.append(newRwd)
+
```
Comment (amayer5125) [inline pkg/bingDashboardParser.py]: Remove extra line here

File: pkg/bingDashboardParser.py [hunk 3]
```diff
@@ -101,124 +105,148 @@ def parseDashboardPage(page, bing_url):
...
-    allRewards.append(currentReward)
 
     return allRewards
 
 def checkForHit(currAction, rewardProgressCurrent, rewardProgressMax, searchLink):
     if currAction is not None:
         if rewardProgressCurrent == 0 and rewardProgressMax == 0:
             if currAction.get_text().lower().find('points') != -1:
-                rewardProgressMax = int(currAction.get_text().split(' ')[0])
+                try: rewardProgressMax = int(currAction.get_text().split(' ')[0])
```
Comment (amayer5125) [inline pkg/bingDashboardParser.py]: this should be two lines
```
try:
    rewardProgressMax = int(currAction.get_text().split(' ')[0])
```

Comment (amayer5125) [reply pkg/bingDashboardParser.py]: Same for the next line:
```
except ValueError:
    pass
```

**Comment by specu** [comment]
@amayer5125  (cc @baltimoron  ) I can commit to submitting those cosmetic patches after the merge. I think in the interest of everyone's time that's the most efficient way to go rather than creating a new pull request. the main tree right now is not working at all, all the accounts have been converted.

**Comment by baltimoron** [comment]
@amayer5125 fixed those issues with the latest commit.

**Pull request merged**